core/src/main/java/com/novoda/accessibility/AccessibilityServices.java

```diff
@@ -40,9 +40,26 @@ public boolean isSpokenFeedbackEnabled() {
     }
 
     /**
-     * Reports if video captioning is enabled on the device.
+     * Returns true if video captioning is enabled on the device.
      */
     public boolean isClosedCaptioningEnabled() {
         return captionManager.isClosedCaptioningEnabled();
     }
+
+    /**
+     * Returns true if the Switch Access accessibility service is enabled.
+     */
+    public boolean isSwitchAccessEnabled() {
+        return isAccessibilityServiceEnabled(SWITCH_ACCESS.qualifiedName());
+    }
+
+    private boolean isAccessibilityServiceEnabled(String serviceName) {
+        List<AccessibilityServiceInfo> enabledServices = getEnabledServicesFor(AccessibilityServiceInfo.FEEDBACK_ALL_MASK);
```

Comment:
this is using FEEDBACK_ALL, so it doesn't matter what type of accessibility service is: if it's enabled, it'll grab it